src/assets/scripts/math/circle.js

```diff
@@ -64,3 +64,27 @@ export const fix_angle = (radians) => {
 
     return radians;
 };
+
+/**
+ * normalize angles to within 0° - 360°
+ * @param  {number} degrees an angle
+ * @return {number}         an angle within [0,360]
+ */
+export const degrees_normalize = (degrees) => {
+    if (degrees < 0) {
+        return 360 + (degrees % 360);
+    }
+    else return degrees % 360;
+};
+
+/**
+ * normalize angles to within 0 - 2π
```

Comment:
the main reason is because [YuiDoc](http://yui.github.io/yuidoc/syntax/index.html) and [JSDOC](http://usejsdoc.org/howto-es2015-classes.html).  Both of these things output api documentation based on comment blocks.  See [StrutureJs](http://codebelt.github.io/StructureJS/docs/) written by some people I work with, for an example of what that actually means.  All the documentation there was generated from comment blocks, just like the ones I'm using here.

I tend to lean more towards the YuiDoc syntaxes, but they are mostly the same.